src/DIRAC/DataManagementSystem/Agent/FTS3Agent.py

```diff
@@ -254,7 +251,11 @@ def _monitorJob(self, ftsJob):
             res = self.fts3db.updateJobStatus(upDict)
 
             if ftsJob.status in ftsJob.FINAL_STATES:
-                self.__sendAccounting(ftsJob)
+                # Send to Accounting by default
+                if self.monitoringOption == "Monitoring":
+                    self._sendMonitoring(ftsJob)
+                else:
+                    self.__sendAccounting(ftsJob)
```

Comment:
If you use my suggestion above, then here it would become:
```suggestion
                if "Monitoring" in self.monitoringOptions:
                    self._sendMonitoring(ftsJob)
                if "Accounting" in self.monitoringOptions:
                    self.__sendAccounting(ftsJob)
```